README.md

```diff
@@ -1,236 +1,165 @@
+# Final group capstone - Doctor Appointment Booking  App 
```

Comment:
- [ ] Your readme file has some formatting/rendering issues:

![image](https://github.com/Benawi/Doctor_Appointment-Booking_App/assets/57421823/4bc27833-b952-4568-9ec5-3a77f858c63c)

![image](https://github.com/Benawi/Doctor_Appointment-Booking_App/assets/57421823/c0b5715b-a647-497c-bc8f-ff32b81d0500)


Kindly correct the formatting so your readme renders correctly. 👍